TombEngine/Game/Lara/lara_struct.h

```diff
@@ -825,6 +825,25 @@ enum LaraAnim
 };
 #pragma endregion
 
+enum LaraExtraAnim
```

Comment:
As I said on discord, make sure to correctly format enum names :)

Reply:
Fixed.